### DISCUSSION TITLE
How to backup project via backend

### DISCUSSION THREAD

**Discussion by quangtrongonline** [wintercms/winter#873] [Q&A]
I am wanting to backup all the files in my project. But I don't have access to ssh. I can only access the backend. Is there a way to backup the project via the backend?

**Comment by bennothommo** [comment]
@quangtrongonline your hosting provider should be able to provide you with backups. Most hosting providers use cPanel which has a backup feature. I don't believe there's a Winter plugin available for backups yet, and the only one in October is paid.

**Reply by quangtrongonline** [reply]
@bennothommo My project is running on a vps, however i have lost control on this vps. Can I back up my project with php code or something in the backend? Where all my files and folders are chmoded as 777

**Comment by damsfx** [comment]
Maybe this plugin can help : https://github.com/ad-sharky/wn-backup-plugin

**Reply by quangtrongonline** [reply]
@damsfx is this plugin installable via wintercms backend?

**Reply by damsfx** [reply]
I don't think. The Winter's marketplace is still a pending future.

**Reply by quangtrongonline** [reply]
In my case i don't have ssh permission. So I can't install it using composer or artisan

**Reply by bennothommo** [reply]
Do you have FTP access? If so, install the plugin locally using Composer and then copy the files across to your server.

**Reply by quangtrongonline** [reply]
I don't have FTP access, I can only access the backend

**Reply by bennothommo** [reply]
Unfortunately you are out of luck then. Sorry :(

**Reply by quangtrongonline** [reply]
Thank you, i hope someone has a good solution for me in this case :(

**Reply by LukeTowers** [reply]
You can write PHP code in the PHP section of CMS pages and then execute it by running the pages in the browser, but if you have safe mode enabled you might be SOL.

**Reply by quangtrongonline** [reply]
I also thought about this solution, i will try it. Thank you!
